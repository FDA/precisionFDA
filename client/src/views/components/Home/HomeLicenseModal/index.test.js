import React from 'react'
import { shallow } from 'enzyme'

import { HomeLicenseModal } from './index'


describe('HomeLicenseModal', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<HomeLicenseModal />)

    expect(wrapper).toMatchSnapshot()
  })
})
