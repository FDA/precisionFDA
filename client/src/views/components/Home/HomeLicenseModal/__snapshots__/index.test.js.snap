// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLicenseModal matches snapshot 1`] = `
<div
  className="objects-actions-modal"
>
  <Modal
    hideModalHandler={[Function]}
    modalFooterContent={
      <SwitchFooter
        actionLink=""
        actionType="detach"
        hideAction={[Function]}
      />
    }
    title="Detach License"
  >
    <div>
      <div>
        Are you sure you want to detach the license 
        <span
          style={
            Object {
              "fontStyle": "italic",
            }
          }
        />
        ?
      </div>
    </div>
  </Modal>
</div>
`;
